lib/utils.js

```diff
@@ -21,3 +21,13 @@ export function deprecated (fn, message) {
 
   return newFn
 }
+
+export function printAndExit (message, code = -1) {
```

Comment:
That's to indicate an error. I hope usually, we want to exit because of an error.
That's why I set it as default.